protostar/commands/test/test_case_runners/test_case_runner_factory.py

```diff
@@ -14,16 +14,19 @@
 from protostar.commands.test.test_case_runners.test_case_runner import TestCaseRunner
 from protostar.commands.test.test_config import TestMode
 from protostar.commands.test.test_suite import TestCase
+from protostar.protostar_exception import ProtostarException
 
 
 class TestCaseRunnerFactory:
     def __init__(self, state: TestExecutionState) -> None:
         self._state = state
 
-    def make(self, test_case: TestCase) -> TestCaseRunner:
+    def make(self, test_case: TestCase, profiling=False) -> TestCaseRunner:
         mode = self._state.config.mode
 
         assert mode, "Test mode should be determined at this point."
+        if mode is TestMode.FUZZ and profiling:
+            raise ProtostarException("You cannot profile fuzz tests")
```

Comment:
```suggestion
            raise ProtostarException("Fuzz tests cannot be profiled yet.")
```

Reply:
I am not sure if they will be at any point being honest, I don't see the need for changing the message

Reply:
I am mostly objecting the use of word `you` 😅 `Fuzz tests cannot be profiled.` is also fine.

Reply:
Fair point

Reply:
Done